apps/ledger-live-mobile/src/screens/OperationDetails/Content.tsx

```diff
@@ -452,7 +454,12 @@ export default function Content({
         </View>
       ) : null}
 
-      <Extra extra={extra} type={type} account={account} />
+      <Extra
+        operation={operation}
```

Comment:
Adding `operation` so that we can display the `operation.value` for `STAKE` operations in the operation details view:

<img width="299" alt="image" src="https://user-images.githubusercontent.com/8867651/203608870-c9d8b3a9-743d-41a0-a355-22ebd4982dd5.png">